src/ui/widgets/Line/line.tsx

```diff
@@ -28,30 +33,95 @@ export const LineComponent = (props: LineComponentProps): JSX.Element => {
   const {
     visible = true,
     transparent = false,
-    backgroundColor,
+    backgroundColor = Color.fromRgba(0, 0, 255),
     rotationAngle = 0,
     width,
-    lineWidth = 1
+    height,
+    lineWidth = 1,
+    points,
+    arrowLength = 2,
+    arrows = 0
   } = props;
 
-  const styleProps = {
-    backgroundColor: transparent ? Color.TRANSPARENT : backgroundColor,
-    visible
-  };
+  const transform = `rotation(${rotationAngle},0,0)`;
 
-  const shapeProps = {
-    shapeWidth: `${width}px`,
-    shapeHeight: `${lineWidth}px`
-  };
+  // Each marker definition needs a unique ID or colours overlap
+  // Get random decimal number, take decimal part and truncate
+  const uid = String(Math.random()).split(".")[1].substring(0, 6);
```

Comment:
There is an npm library that could do this called uuid: https://www.npmjs.com/package/uuid. Looks fairly easy to use. Not that there is anything wrong with this method, just wondering if using the library would make it more transparent although it would add another dependency.

Reply:
I was hesitant to add another dependency, but I think it's definitely worth it for transparency. I've added that in now.